src/soundio/soundmanagerutil.cpp

```diff
@@ -154,7 +154,7 @@ QString AudioPath::getStringFromType(AudioPathType type) {
         // handle this -- bkgood
         return QStringLiteral("Invalid");
     case MASTER:
-        return QStringLiteral("Master");
+        return QStringLiteral("Main");
```

Comment:
Since this is the programmatic name, I am afraid this likely breaks the user config.

Reply:
okay I see the comment above the function explaining that now